spring-cloud-dataflow-shell-core/src/main/java/org/springframework/cloud/dataflow/shell/command/ConfigCommands.java

```diff
@@ -234,25 +245,15 @@ public String info() {
 	}
 
 	@Override
-	public void onApplicationEvent(ApplicationReadyEvent event) {
-		//Only invoke if the shell is executing in the same application context as the data flow server.
+	public void onApplicationEvent(ApplicationEvent event) {
 		if (!initialized) {
-			target(this.serverUri, this.userName, this.password, this.skipSslValidation);
-		}
-	}
-
-	@Override
-	public void afterPropertiesSet() throws Exception {
-		//Only invoke this lifecycle method if the shell is executing in stand-alone mode.
```

Comment:
My bad, I realize now that you even removed the `afterPropertiesSet ()` method. But then, isn't `initialized` always false?